spec/data-model/README.md

```diff
@@ -237,19 +234,16 @@ interface VariableRef {
 A `FunctionAnnotation` represents a _function_ _annotation_.
 The `name` does not include the `:` starting sigil.
 
-Each _option_ is represented by an `Option`.
+Each _option_ is represented by an `Options` key-value mapping.
 
 ```ts
 interface FunctionAnnotation {
   type: "function";
   name: string;
-  options: Option[];
+  options: Options;
 }
 
-interface Option {
-  name: string;
-  value: Literal | VariableRef;
-}
+type Options = Map<string, Literal | VariableRef>;
```

Comment:
Put documentation here for both `FunctionAnnotation` and Markup. (not using suggestion markdown because structural change would be needed)

> Options is a key-value map containing options.
> Implementations can use an ordered map if they want 
> to maintain the source ordering of options.
> Implementations might also canonicalize messages by ordering
> options into code point order.